include/cell.h

```diff
@@ -222,6 +224,31 @@ class Cell {
                                     unsigned phase, double pvolume,
                                     const Eigen::Matrix<double, 6, 1>& pstress);
 
+  //! Assign velocity constraint
+  //! Directions can take values between 0 and Dim-1
+  //! \param[in] face_id Face of cell of velocity constraint
+  //! \param[in] dir Direction of velocity constraint
+  //! \param[in] velocity Applied velocity constraint
+  bool assign_velocity_constraint(unsigned face_id, unsigned dir,
+                                  double velocity);
+
+  //! Apply velocity constraints
+  void apply_velocity_constraints();
+
+  //! Compute normal vector
+  void compute_normals();
+
+  //! Return number of normal vectors
+  //! \retval number of normal vectors
+  unsigned nnormal() const { return face_normals_.size(); }
+
+  //! Return unit normal vector, positive pointing outside of the element
+  //! \param[in] face_id of constraint
+  //! \retval unit normal vector
+  Eigen::VectorXd normal(unsigned face_id) const {
```

Comment:
You don't need this function, if only `cell` is going to use it. It would be a lot clearer to just access the private variable.

Reply:
I think I will use it later. Either way, we need to check them right? How do we test them without this function?

Reply:
Please don't include functions, which you are not using. This function is not needed, you can get the face_normal directly within the cell. I can't see it being used outside the class.